pkg/apis/eksctl.io/v1alpha4/types.go

```diff
@@ -128,6 +130,13 @@ var (
 
 	// DefaultNodeSSHPublicKeyPath is the default path to SSH public key
 	DefaultNodeSSHPublicKeyPath = "~/.ssh/id_rsa.pub"
+
+	// DefaultAvailabilityZones for the DefaultRegion
+	DefaultAvailabilityZones = []string{
```

Comment:
Please consider using `pkg/az`, and do feel free to suggest any changes to that. E.g., you might want to have deterministic behaviour, as it currently pick AZs randomly. But let's not add another AZ selection mechanism outside of that package.

Reply:
this one works, and is actually missing, because you have a `DefaultRegion`, `pkg/az` is dufferent story

Reply:
also I need this code have offline capability if requested (for UX/DX reasons)